com.unity.render-pipelines.core/Editor/AdditionalPropertiesPreferences.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEditor.SceneManagement;
+using UnityEditorInternal;
+
+namespace UnityEditor.Rendering
+{
+    class AdditionalPropertiesPreferences
+    {
+        class Styles
+        {
+            public static readonly GUIContent additionalPropertiesLabel = new GUIContent("Additional Properties", "Toggle all additional properties to either visible or hidden.");
```

Comment:
@MaddalenaUnity the problem is that I think we changed the behavior. It's no longer a default but rather a toggle so that's why I changed the tool tip. Talking about default here might make people think that it's what they should expect when opening an inspector for the first time but it does not make sense anymore since the state is per type and no longer per object.

Reply:
I double checked with @MaddalenaUnity is ok to go with this tooltip. Thank you @JulienIgnace-Unity